test/testdrive/indexes.td

```diff
@@ -203,6 +203,13 @@ contains:Cannot specify both FROM and ON
 ! SHOW INDEXES FROM nonexistent
 contains:unknown schema 'nonexistent'
 
+> CREATE TABLE bar ();
+> CREATE INDEX bar_ind ON bar ();
+
+> SHOW INDEXES
+bar_ind bar <VARIABLE_OUTPUT> <null>
```

Comment:
Wait, I thought we wanted an empty array as output, not NULL?

Reply:
That was the original plan, but since the default value for `ARRAY_AGG` when there's no rows is `NULL`, I thought that maybe it made more sense to have `NULL` here.

I'm open to trying to switch it back to an empty array here, though it might make the query even more complicated and slow.

Reply:
TIL about `coalesce`, I just switched this to default to an empty array.